.travis.yml

```diff
@@ -6,5 +6,8 @@ otp_release: '20.0'
 addons:
   postgresql: "9.6"
 
+env:
+  - MIX_ENV=test
+
 script:
-  - mix test
+  - mix coveralls.travis
```

Comment:
Is that task enough? We don't need to run the regular `test` task? I'm asking because I haven't used coveralls before.

Reply:
Yes, this is the command suggested in the coveralls Readme. It does all the job, runs the regular tests, tracks the coverage and send to travis. The CI build failed because we have to turn on the service for the repository, similarly to Travis, in order to get the results published. Here is the link for [Coveralls page](https://coveralls.io/)

Reply:
Yup it is, we use it in benchee. It seems the build is failing because it needs to be setup on coveralls from someone who has repo access I believe.